infinispan-client/src/test/java/io/quarkus/ts/openshift/infinispan/client/InfinispanGreetingResourceOpenShiftIT.java

```diff
@@ -0,0 +1,97 @@
+package io.quarkus.ts.openshift.infinispan.client;
+
+import io.fabric8.openshift.client.OpenShiftClient;
+import io.quarkus.ts.openshift.common.AdditionalResources;
+import io.quarkus.ts.openshift.common.Command;
+import io.quarkus.ts.openshift.common.CustomizeApplicationDeployment;
+import io.quarkus.ts.openshift.common.CustomizeApplicationUndeployment;
+import io.quarkus.ts.openshift.common.ManualApplicationDeployment;
+import io.quarkus.ts.openshift.common.OnlyIfConfigured;
+import io.quarkus.ts.openshift.common.OpenShiftTest;
+import org.junit.jupiter.api.MethodOrderer;
+import org.junit.jupiter.api.Order;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestMethodOrder;
+
+import java.io.IOException;
+import java.nio.charset.Charset;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import static io.restassured.RestAssured.when;
+import static org.hamcrest.CoreMatchers.is;
+
+@OpenShiftTest
+@ManualApplicationDeployment
+@AdditionalResources("classpath:clientcert_secret.yaml")
+@OnlyIfConfigured("ts.operator-installed")
```

Comment:
Maybe this property should be named differently, like `include.datagrid`? Our other test module that requires operators has a property like that. See
https://github.com/quarkus-qe/quarkus-openshift-test-suite/blob/ac303ee97f1a98bbb26fc32f69f61e871c883635/pom.xml#L315

Also, maybe it could be done through Maven module included by profile, especially considering that in the future there are more tests planned to be implemented :)